examples/logs-slack-alerts/main.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import requests
+from google.cloud import bigquery
+
+
+logging.getLogger().setLevel(logging.INFO)
+
+#Fill in variables with your project setup
+variables ={
```

Comment:
Anything configurable should be read from an environment variable

Reply:
Done